<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SimpleAddIMAccount</class>
 <widget class="QDialog" name="SimpleAddIMAccount" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WengoPhone - Add a Contact Account</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="groupComboBox" />
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="nameLineEdit" />
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="contactIdLineEdit" />
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="contactTypeComboBox" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Group</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Contact Id</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Contact type</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="advancedConfigPushButton" >
       <property name="text" >
        <string>Advanced</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>contactTypeComboBox</tabstop>
  <tabstop>contactIdLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>groupComboBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>advancedConfigPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
